### PR TITLE
use monotonically increasing IDs for the labels

### PR DESCRIPTION
**Pull request by notwaldorf**
Fixes https://github.com/PolymerElements/paper-input/issues/113

Most of the changes are indentation; use https://github.com/PolymerElements/paper-input/pull/356/files?w=1 to ignore whitespace changes.

### CODE REVIEW

File: test/paper-input.html [hunk 1]
```diff
@@ -274,11 +283,24 @@
...
-        assert.equal(input.inputElement.getAttribute('aria-labelledby'), Polymer.dom(input.root).querySelector('label').id, 'aria-labelledby points to the label');
+      test('has aria-labelledby, which is monotonically increasing', function() {
+        var inputs = fixture('multiple-inputs');
+
+        // Find the first index of the input in this fixture. Since the label
+        // ids monotonically increase every time a new input is created, and
+        // this fixture isn't the first one in the document, we're going to start
+        // at an ID > 1.
+        var firstLabel = Polymer.dom(inputs[0].root).querySelector('label').id;
+        var index = firstLabel.substr(firstLabel.lastIndexOf('-') + 1)
```
Comment (bicknellr) [inline test/paper-input.html]: Maybe `parseInt` this for good measure?

**Comment by bicknellr** [comment]
LGTM

**Comment by bicknellr** [comment]
Oh, this PR would probably be a good place to fix [add-on ID generation](https://github.com/PolymerElements/paper-input/blob/master/paper-input-behavior.html#L422) (mentioned in the bug) too.

**Comment by notwaldorf** [comment]
@bicknellr Done! 👍

File: test/paper-input.html (hunk 1 above)
Comment (bicknellr) [inline test/paper-input.html]: Missing a semicolon too.

**Comment by bicknellr** [comment]
Cool, still LGTM.

**Pull request merged**